src/rpc/request.cpp

```diff
@@ -26,6 +26,17 @@
  *
  * 1.0 spec: http://json-rpc.org/wiki/specification
  * 1.2 spec: http://jsonrpc.org/historical/json-rpc-over-http.html
+ *
+ * If the server receives a request with the JSON-RPC 2.0 marker `{"jsonrpc": "2.0"}`
+ * then Bitcoin will respond with a strictly specified response.
+ * It will only return an HTTP error code in response to HTTP errors like if
```

Comment:
In commit "doc: add comments and release-notes for JSON-RPC 2.0" (756da6d1e67fba65a992dc0090ee8c0cfa26abe3)

Maybe replace "errors like if..." with "errors in cases like the endpoint not being found (404) or the RPC request not being formatted correctly (500)." Grammar sounds a little off the way it is written now.